features/static_versions_post_with_id.feature

```diff
@@ -0,0 +1,328 @@
+                    "distributions": [
+                        {
+                            "title": "Published Dataset CSV",
+                            "format": "csv",
+                            "media_type": "text/csv",
+                            "download_url": "/downloads/datasets/static-dataset-1/editions/2024/versions/1.csv",
+                            "byte_size": 150000
+                        }
+                    ]
+                },
+                {
+                    "id": "static-version-unpublished",
+                    "edition": "2025",
+                    "edition_title": "2025 Edition",
+                    "links": {
+                        "dataset": {
+                            "id": "static-dataset-1"
+                        },
+                        "edition": {
+                            "href": "/datasets/static-dataset-1/editions/2025",
+                            "id": "2025"
+                        },
+                        "version": {
+                            "href": "/datasets/static-dataset-1/editions/2025/versions/2",
+                            "id": "2025"
+                        },
+                        "self": {
+                            "href": "/datasets/static-dataset-1/editions/2025/versions/2"
+                        }
+                    },
+                    "version": 2,
+                    "release_date": "2025-01-01T09:00:00.000Z",
+                    "state": "associated",
+                    "type": "static",
+                    "distributions": [
+                        {
+                            "title": "Unpublished Dataset CSV",
+                            "format": "csv",
+                            "media_type": "text/csv",
+                            "download_url": "/downloads/datasets/static-dataset-1/editions/2025/versions/2.csv",
+                            "byte_size": 150000
+                        }
+                    ]
+                }
+            ]
+            """
+    
+    Scenario: Successfully creating a new version
+        Given private endpoints are enabled
+        And I am identified as "user@ons.gov.uk"
+        And I am authorised
+        When I POST "/datasets/static-dataset-1/editions/2024/versions/2"
+            """
+            {
+                "release_date": "2024-12-01T09:00:00.000Z",
+                "edition_title": "2024",
+                "distributions": [
+                    {
+                        "title": "Full Dataset CSV",
+                        "format": "csv",
+                        "media_type": "text/csv",
+                        "download_url": "/downloads/datasets/static-dataset-1/editions/2024/versions/2.csv",
+                        "byte_size": 100
+                    }
+                ]
+            }
+            """
+        Then the HTTP status code should be "201"
+        And I should receive the following JSON response ignoring last updated:
```

Comment:
I'm not sure why a new step had to be created to ignore the last updated value - we have examples for setting the last_updated field so they match when the "I should receive the following JSON response" step is called.  

Example in dataset-api - https://github.com/ONSdigital/dp-dataset-api/blob/develop/features/steps/steps.go#L483